.gitignore

```diff
@@ -8,11 +8,25 @@
 source/plugins/go/src/profiling
 .vscode/launch.json
 source/plugins/go/src/vendor/
-# go shared object files
+# ignore go packages
+source/plugins/go/pkg/
+# ignore all these types
 *.so
 *.zip
+*.exe
+*.jar
+*.dll
+*.obj
 # .net code build artificats
 build/windows/installer/certificategenerator/bin
 build/windows/installer/certificategenerator/obj
 # files under omsagentwindows dir since this temp directory to build the docker image
-kubernetes/windows/omsagentwindows
\ No newline at end of file
+kubernetes/windows/omsagentwindows
+# ignore everything under intermediate directory
+intermediate
+# ignore all intermeditae *.pyc files
+*.pyc
```

Comment:
move all file extensions together in one sequence

Reply:
> move all file extensions together in one sequence

Fixed.